### PR TITLE
Auction Reporting

### PR DESCRIPTION
**Pull request by murphys0709**

### CODE REVIEW

**Review by Brandr0id — COMMENTED** [review]
What about just 'Auction Reporting.md' for the filename?

Should we also update the contents or link from other locations to help guide readers to this flow?

File: Auction Reporting.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Auction Reporting
+## Overview
+In Microsoft Edge, we are supporting a limited version of ads reporting. For this limited version of reporting, reports will be per-event. Future work includes replacing these event-level reports with aggregated reporting.
+
+Edge is only supporting ad auctions that are hosted by the seller’s front end (SFE), within a Trusted Execution Environment (TEE). When auctions are completed, two types of reports are generated via User Defined Functions (UDF):
```
Comment (erik-anderson) [inline Auction Reporting.md:5]: Nit: "User Defined Functions" -> "user-defined functions" and "UDF" -> "UDFs".

File: Auction Reporting.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+-   Seller Reports on Auction Results – generated by the function reportResults()
+-   Buyer Report on Auction Winner – generated by the function  reportWin()
+    
+
+The output of the UDFs is a report in the format of [a reporting URL.](https://github.com/privacysandbox/bidding-auction-servers/blob/4a7accd09a7dabf891b5953e5cdbb35d038c83c6/api/bidding_auction_servers.proto#L96C26-L96C26) For example:
+
+    'https://buyer-reporting-server.example/reporting?bid=${bidInfo}
+
+Additionally, reporting is available for interactions within embedded frames. These reports are generated by the reportEvent() API.
```
Comment (erik-anderson) [inline Auction Reporting.md:15]: Nit: add blank line after this one.

File: Auction Reporting.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+# Buyer and Seller Reports
+## Data from the Browser
+The seller’s ad service receives the following data from the browser:
+
+-   A [Request Blob](https://github.com/WICG/turtledove/blob/36bf1d00641d5f42850ad6f97a64324b53b47c70/FLEDGE_browser_bidding_and_auction_API.md#request-blob-format), contains the following information:
```
Comment (erik-anderson) [inline Auction Reporting.md:20]: Nit: prefer not Title Casing things that aren't a proper name. "Request Blob" -> "request blob"

Comment (erik-anderson) [inline Auction Reporting.md:20]: Nit: add the word "which" before "contains."

File: Auction Reporting.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    -   Properties: The version and compression type used for the message.
+    -   Publisher: String listing the publisher site
+    -   Generation ID: Random identifier produced by the browser that the server may use for rate limiting
+    -   Interest Groups: Encoded and compressed list of interest groups.
```
Comment (erik-anderson) [inline Auction Reporting.md:24]: Same not-title-casing feedback; "Interest Groups" -> "Interest groups".

Comment (erik-anderson) [reply Auction Reporting.md:24]: Also applies to line 26.

File: Auction Reporting.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+||buyerReportingId|If set, the value is used instead of the interest group name for reporting in reportWin. Note that this field needs to be jointly k-anonymous with the interest group owner, bidding script URL, and render URL to be provided to these reporting functions (in the same way that the interest group name would have needed to be).|
+||buyerAndSellerReportingId|If set, the value is used instead of the interest group name or buyerReportingId for reporting in reportWin and reportResult. Note that this field needs to be jointly k-anonymous with the interest group owner, bidding script URL, and render URL to be provided to these reporting functions (in the same way that the interest group name would have needed to be).|
+
+The report URL can contain any of the information above as a param, so long as the URL meets the following requirements:
+- It is a valid HTTPS URL.  
+- It is no longer than the browser’s URL length limits.
+- It has been attested to the Ad Selection API.  
+
+Each UDF can only send a single report URL.  If multiple attempts are made, an exception will be thrown.
```
Comment (erik-anderson) [inline Auction Reporting.md:65]: Nit: blank line after this one.

File: Auction Reporting.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+## Currency in Reporting:
+An optional mode is available that converts all bid-related information to the seller's preferred currency (in component auctions, for that component's seller). This is configured via the sellerCurrency setting in each auction configuration.  More details on the sellerCurrency  option are available [here](https://github.com/WICG/turtledove/blob/2ec2eac68b8c0218d667828c508a788096205dd6/FLEDGE.md#53-currencies-in-reporting).
```
Comment (erik-anderson) [inline Auction Reporting.md:67]: Double space between "sellerCurrency" and "option"; change to a single space.

**Review by erik-anderson — CHANGES_REQUESTED** [review]

**Review by erik-anderson — APPROVED** [review]

**Pull request merged**